stress_test.py

```diff
@@ -287,13 +289,13 @@ def main():
     if args.heavy:
         print('Setting the heavy configuration...')
         args.large_file_size_gb = 20
-        args.bundle_upload_count = 1000
-        args.create_worksheet_count = 1000
+        args.bundle_upload_count = 500
+        args.create_worksheet_count = 500
         args.parallel_runs_count = 500
-        args.large_docker_runs_count = 2000
-        args.test_infinite_memory = True
-        args.test_infinite_gpu = True
-        args.infinite_gpu_runs_count = 100
+        args.large_docker_runs_count = 200
+        args.test_infinite_memory = False
```

Comment:
@epicfaace I updated the numbers and added `--heavy`. I will bring back the infinite runs in the near future.